index.js

```diff
@@ -34,6 +37,10 @@ console.log('hello world')
 
 // TODO: ADD CODE HERE
 
+const firstName = "Dejan";
+const lastName = "Pantelic";
+console.log(firstName + " " + lastName);
```

Comment:
You could also use a template literal here:

```javascript
`${firstName} ${lastName}`
```

Reply:
I will try to force ES2015 from now on....
Thank you.